docs/docs/cloud/quick_start.md

```diff
@@ -175,36 +213,26 @@ curl --request POST \
         "messages": [
             {
                 "role": "user",
-                "content": "How are you?"
+                "content": "What is the weather in NYC?"
             }
         ]
-    },
-    "metadata": {},
-    "config": {
-        "configurable": {}
-    },
-    "multitask_strategy": "reject",
-    "stream_mode": [
-        "values"
-    ]
+    }
 }'
 ```
 
 If you get back a valid response, then all is functioning properly!
```

Comment:
Ideally this shows an example of what the output might be?

I.e., you should get something like...